.env.example

```
@@ -20,9 +20,13 @@ GOOGLE_CLIENT_SECRET=""
 
 # Prisma
 # https://www.prisma.io/docs/reference/database-reference/connection-urls#env
-DATABASE_URL="postgresql://postgres:strongpassword@localhost:5432/cinemate"
+DATABASE_URL="postgresql://postgres:postgres@127.0.0.1:54322/postgres"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)